src/libs/navigateAfterOnboarding.ts

```diff
@@ -1,22 +1,32 @@
 import ROUTES from '@src/ROUTES';
 import Navigation from './Navigation/Navigation';
 import shouldOpenOnAdminRoom from './Navigation/shouldOpenOnAdminRoom';
-import * as ReportUtils from './ReportUtils';
+import {findLastAccessedReport, isConciergeChatReport} from './ReportUtils';
 
-const navigateAfterOnboarding = (isSmallScreenWidth: boolean, canUseDefaultRooms: boolean | undefined, onboardingPolicyID?: string, activeWorkspaceID?: string) => {
+const navigateAfterOnboarding = (
+    isSmallScreenWidth: boolean,
+    canUseDefaultRooms: boolean | undefined,
+    onboardingPolicyID?: string,
+    activeWorkspaceID?: string,
+    onboardingAdminsChatReportID?: string,
+    shouldOpenAdminRoom?: boolean,
+) => {
     Navigation.dismissModal();
 
     // When hasCompletedGuidedSetupFlow is true, OnboardingModalNavigator in AuthScreen is removed from the navigation stack.
     // On small screens, this removal redirects navigation to HOME. Dismissing the modal doesn't work properly,
     // so we need to specifically navigate to the last accessed report.
     if (!isSmallScreenWidth) {
+        if (shouldOpenAdminRoom && onboardingAdminsChatReportID) {
+            Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(onboardingAdminsChatReportID));
+        }
         return;
     }
 
-    const lastAccessedReport = ReportUtils.findLastAccessedReport(!canUseDefaultRooms, shouldOpenOnAdminRoom(), activeWorkspaceID);
+    const lastAccessedReport = findLastAccessedReport(!canUseDefaultRooms, shouldOpenOnAdminRoom(), activeWorkspaceID);
     const lastAccessedReportID = lastAccessedReport?.reportID;
     // we don't want to navigate to newly creaded workspace after onboarding completed.
```

Comment:
```suggestion
    // we don't want to navigate to newly created workspaces after onboarding is completed.
```
Just noticed this, might as well fix.